cmd/validate/validate.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021 k0s authors
```

Comment:
The copyright linter complains about this https://github.com/k0sproject/k0s/blob/main/hack/copyright.sh#L23-L25

We could either remove the -1 flag from git log in the copyright script  and then it works, or we can update the copyright to 2025.

Reply:
Damn. Did a force push and forgot to add the local change first ... that's what you get when you're in a hurry ... :disappointed: